lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -55,15 +55,19 @@ def load_courses(self):
         # Add a course with a unicode name, if the modulestore
         # supports adding modules.
         if hasattr(store, 'create_xmodule'):
```

Comment:
According to the coverage report, split does not have a "create_xmodule" method. Should I change this to something else?